node/src/test/tests.rs

```diff
@@ -167,6 +168,50 @@ fn test_inventory_sync() {
     assert_eq!(a, b);
 }
 
+#[test]
+fn test_tracking() {
+    let rng = fastrand::Rng::new();
+    let mut alice = Peer::config(
+        "alice",
+        Config {
+            project_tracking: ProjectTracking::Allowed(HashSet::with_hasher(rng.clone().into())),
```

Comment:
FYI you can just do `HashSet::default()` now, when you don't care about reproducibility